2018-05-15T07:29:35.817Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.819Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'nga@glorious.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.910Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.911Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'phongvu@phongvu.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.914Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper2@shipperltd2.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.914Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper2@shipperltd2.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper2@shipperltd2.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.921Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hoanglong@hoanglong.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.921Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hoanglong@hoanglong.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'hoanglong@hoanglong.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.922Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.922Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'tech@techcombank.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.934Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.934Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'linh@vinhsang.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:35.944Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper1@shipperltd1.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:35.944Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper1@shipperltd1.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'shipper1@shipperltd1.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:29:36.135Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:29:36.136Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:31:46.827Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'bob@pchardwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:46.828Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'bob@pchardwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'bob@pchardwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.687Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'jojo@innovativesolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.692Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'jojo@innovativesolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'jojo@innovativesolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.727Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'mary@iseriesexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.727Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'mary@iseriesexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'mary@iseriesexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.809Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'info@pchardwarenowinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.809Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'info@pchardwarenowinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'info@pchardwarenowinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.841Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'wilber@coolingsystemsrusinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.841Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'wilber@coolingsystemsrusinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'wilber@coolingsystemsrusinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.943Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'online@theappstoreinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.943Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'online@theappstoreinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'online@theappstoreinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.950Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'aiesha@cloudninesoftwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.950Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'aiesha@cloudninesoftwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'aiesha@cloudninesoftwareinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.980Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '003' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.980Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '003' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '003' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:47.985Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:47.985Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.004Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'Erin@2ndlifesystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.004Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'Erin@2ndlifesystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'Erin@2ndlifesystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.013Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'joshua@officeexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.014Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'joshua@officeexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'joshua@officeexpertsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.153Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.153Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.228Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'eric@bornonthecloudinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.228Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'eric@bornonthecloudinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'eric@bornonthecloudinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.287Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'abdul@cognitiveadvantageinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.287Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'abdul@cognitiveadvantageinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'abdul@cognitiveadvantageinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.295Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.295Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.323Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.323Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.405Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.405Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.411Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'alwayson@nonstopinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.411Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'alwayson@nonstopinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'alwayson@nonstopinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.419Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'buyer1@hybridclouddesignsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.419Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'buyer1@hybridclouddesignsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'buyer1@hybridclouddesignsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.561Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'power@upssystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.561Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'power@upssystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'power@upssystemsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Provider' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.598Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '005' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.598Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '005' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '005' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.602Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'abby@kidfriendlyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.602Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'abby@kidfriendlyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'abby@kidfriendlyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.615Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.615Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.682Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'anytime@virtualpaperinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.682Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'anytime@virtualpaperinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'anytime@virtualpaperinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Seller' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.722Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.723Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'easymoney@easymoneyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.FinanceCo' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.775Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'speedmatters@fasteddyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.775Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'speedmatters@fasteddyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'speedmatters@fasteddyinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:48.916Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'kristen@gameify.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:48.917Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'kristen@gameify.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'kristen@gameify.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:49.008Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:49.008Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:49.011Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'yes@softwaresolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:49.012Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'yes@softwaresolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'yes@softwaresolutionsinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Buyer' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:49.045Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:49.045Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'noop@dummy' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:49.063Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID 'wantitnow@overnightexpressinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:49.064Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'wantitnow@overnightexpressinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID 'wantitnow@overnightexpressinc.com' in collection with ID 'Participant:org.acme.Z2BTestNetwork.Shipper' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-15T07:31:55.219Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-15T07:31:55.219Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-15T07:31:56.030Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '1ef2bb12d0976801df2e76ae5deac4fd44e9e3bf14a3295f6369830b13bcee2d' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '1ef2bb12d0976801df2e76ae5deac4fd44e9e3bf14a3295f6369830b13bcee2d' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:56.030Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction 'ef57297f106f959c9ef887d96be836eae1d4977af771eb0bec9eb61b45c93b81' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction 'ef57297f106f959c9ef887d96be836eae1d4977af771eb0bec9eb61b45c93b81' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:56.030Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction 'a68c565832849f3b7e70c1397b0d06a6dee30b1667c9afc32c89ea92589eec88' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction 'a68c565832849f3b7e70c1397b0d06a6dee30b1667c9afc32c89ea92589eec88' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:56.030Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction 'f5d0cffad24731009301865648abcc65fee5caccec4136f7039a3631ef7d7ba2' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction 'f5d0cffad24731009301865648abcc65fee5caccec4136f7039a3631ef7d7ba2' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:57.623Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '6ec8af11b3bbc8f355be9ab0ef4ade574729cbc6398c764d325590c62824ee9c' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '6ec8af11b3bbc8f355be9ab0ef4ade574729cbc6398c764d325590c62824ee9c' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:57.623Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '93fd8edb9495a9b634f03f87ec88356642d0cebc0f665d329ee1af7e1d0acb69' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '93fd8edb9495a9b634f03f87ec88356642d0cebc0f665d329ee1af7e1d0acb69' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:57.623Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '504954453e4382fd89bd3a2c27a00326a72a5e41672d31aaf91af52a725d87f6' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '504954453e4382fd89bd3a2c27a00326a72a5e41672d31aaf91af52a725d87f6' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:58.782Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '54c6cbd3cc5619f0b57658ab10b14d85aa77043feed188d01a12725a3c72e025' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '54c6cbd3cc5619f0b57658ab10b14d85aa77043feed188d01a12725a3c72e025' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.820Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '315ee2f1116de0145cef991d11db8c0bb42317ec78714d94cbb2cb25775d5270' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '315ee2f1116de0145cef991d11db8c0bb42317ec78714d94cbb2cb25775d5270' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.820Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '5ea5d00d78b9b80a5c229836c37772bd5f1eda5d220d41688d016fc9fc38fc78' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '5ea5d00d78b9b80a5c229836c37772bd5f1eda5d220d41688d016fc9fc38fc78' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.820Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction 'cfe56490899a1921cc79c1d091ba4a356d62fc1be0bc3d817c040925c2c6f6ee' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction 'cfe56490899a1921cc79c1d091ba4a356d62fc1be0bc3d817c040925c2c6f6ee' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.820Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction 'a426cc161b0d07f296187dfbb068d05bf313b47a6166ae57e919d10f33267dc8' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction 'a426cc161b0d07f296187dfbb068d05bf313b47a6166ae57e919d10f33267dc8' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.821Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '3eedfd8dcf3821a879972e4df00eeb47a5c3b1b45e94d9507029007c0e09eea3' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '3eedfd8dcf3821a879972e4df00eeb47a5c3b1b45e94d9507029007c0e09eea3' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:31:59.821Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: Peer has rejected transaction '08365537a3a272291a8540e85645a734b80354abb8e75d6ce45388b404fe2cce' with code MVCC_READ_CONFLICT","stack":"Error: Error trying invoke business network. Error: Peer has rejected transaction '08365537a3a272291a8540e85645a734b80354abb8e75d6ce45388b404fe2cce' with code MVCC_READ_CONFLICT\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:188:7)"}$
2018-05-15T07:37:46.430Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/hoanglong@hoanglong.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/hoanglong@hoanglong.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/hoanglong@hoanglong.com"}}$
